x-pack/plugins/ml/server/models/data_recognizer/modules/metricbeat_system_ecs/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "id": "metricbeat_system_ecs",
+  "title": "Metricbeat System",
+  "description": "Detect anomalies in Metricbeat System data - cpu, diskio, filesystem, memory (ECS)",
```

Comment:
I actually didn't create any diskio or memory jobs, so we should drop those from the description. Can we just make it `Detect anomalies in Metricbeat System data` ?

Reply:
I'll edit the module description to your suggested text,